Rakefile

```diff
@@ -59,8 +59,12 @@ namespace :arm do
         md = REGEN_METADATA[dir.to_sym]
         ar_base_command = "#{OS.windows? ? '' : 'mono '} #{REGEN_METADATA[:autorest_loc]}"
 
-        command = "#{ar_base_command} -i #{md[:spec_uri]} -pv #{md[:version]} -n #{md[:ns]} -pn #{md[:pn].nil? ? dir : md[:pn]} -g Azure.Ruby -o lib"
-        command += " -m #{md[:modeler]}" unless md[:modeler].nil?
+        if md[:spec_uri].end_with?('.json')
+          command = "#{ar_base_command} -i #{md[:spec_uri]} -pv #{md[:version]} -n #{md[:ns]} -pn #{md[:pn].nil? ? dir : md[:pn]} -g Azure.Ruby -o lib"
+          command += " -m #{md[:modeler]}" unless md[:modeler].nil?
+        else
+          command = "#{ar_base_command} #{md[:spec_uri]} --package-version=#{md[:version]} --namespace=#{md[:ns]} --package-name=#{md[:pn].nil? ? dir : md[:pn]} --output-folder=#{File.join(Dir.pwd, 'lib')} --ruby --azure-arm"
```

Comment:
Yes. They have changed it to relative path. So, when you have a URI as https://......./compute/xyz.md and output-folder as lib, it will try to write to https://....../compute/lib location :-(